pkg/virt-launcher/notify-client/notify_test.go

```diff
@@ -63,6 +63,7 @@ var _ = Describe("Notify", func() {
 		var mockDomain *cli.MockVirDomain
 		var mockCon *cli.MockConnection
 		var ctrl *gomock.Controller
+		var e eventCaller
```

Comment:
Can we uniform between `var e eventCaller` or `e := eventCaller{}`? :slightly_smiling_face:

Reply:
+1

Reply:
I changed everything to `var e eventCaller`